libs/account/src/devlive/store.ts

```diff
@@ -2,8 +2,9 @@ import { ConnectContract } from '@thecointech/contract';
 import { getSigner, AccountName } from '@thecointech/signers';
 import { NormalizeAddress } from '@thecointech/utilities/Address';
 import { AccountState, buildNewAccount } from '../state';
+import * as Browser from '../store';
 
-const _devWallets = {} as Record<string, AccountState>;
+const _devWallets = Browser.getAllAccounts();
```

Comment:
include previously saved accounts to make dev:live better testing environment.

`dev` resets all accounts on every run, but dev:live should be close to final environment